ecs.tf

```diff
@@ -32,4 +32,12 @@ resource "aws_ecs_service" "main_service" {
   desired_count              = var.desired_task_count
   launch_type                = var.launch_type
   deployment_maximum_percent = var.service_deployment_maximum_percent
+  
+  dynamic "ordered_placement_strategy" {
+    for_each = var.ordered_placement_strategies
+    content {
+      type  = ordered_placement_strategy.value.type
+      field = ordered_placement_strategy.value.field
+    }
+  }
```

Comment:
Kind of weird seeing for loops in terraform, but cool nonetheless

Reply:
Yeah it seems unfortunate because all I'm trying to do here is pass the `ordered_placement_strategy` from an input variable to the block here. I wasn't able to find a way to do that without declaring a `dynamic` block and looping over the list. If there's a better way to do that, I'm all ears!